app/containers/Contact/styles/colors.js

```diff
@@ -0,0 +1,10 @@
+const colors = {
```

Comment:
Could you move this to app/styles/colors, we can use this color code over entire app. I was thinking about this to be honest :)